Editor/EditorCore/ProBuilderShapeEditor.cs

```diff
@@ -61,11 +110,6 @@ bool HasMultipleShapeTypes
 
         void OnEnable()
         {
-            m_ShapeProperty = serializedObject.FindProperty("m_Shape");
-            m_ShapePivotProperty = serializedObject.FindProperty("m_PivotLocation");
-            m_ShapeSizeXProperty = serializedObject.FindProperty("m_Size.x");
-            m_ShapeSizeYProperty = serializedObject.FindProperty("m_Size.y");
-            m_ShapeSizeZProperty = serializedObject.FindProperty("m_Size.z");
```

Comment:
Why is this not working specifically for `ProBuilderShape`? That's concerning, and smells like a bigger problem than just null checks.

Does this reproduce with other editors? Is it specific to `SerializeReference`?

Reply:
it seems to happen only the first frame on entering playmode,not sure the problem is common I will have to confirm that as it didn't happen on Joel's PC